### ISSUE TITLE
Overwhelmed by issues? Let's have a party!

### ISSUE THREAD

**Issue by schwesig** [AICoE/content-pipeline#27]
**Overview**
Organizing, sanitizing and prioritizing issues in github shouldn't be a burden, let's make it fun.
Let us invite you to our Triage Party and show you the with fun to filter and follow issues, and how we celebrate quality stages and measures.

**Speakers**
@codificat
@schwesig

**What conference(s) are you submitting this proposal to?**
DevConf.cz (#10)

**Project repo link, or other relevant resources**
https://github.com/operate-first/SRE/issues/389

**Link to abstract**
https://docs.google.com/document/d/17K156k3aHQsOsdKziBytn9KDgkKfyjmtxMLPE6PXqLU/edit?usp=sharing

**Was this proposal accepted?**
- [x] Yes
- [ ] No

**Comment by schwesig** [comment]
SUBMITTED 2021-10-22_09-38 (Berlin/DE)
http://cfp.devconf.info/proposal/70/3419/3845

<details>
<summary>
Title
Overwhelmed by issues? Let's have a party!
</summary>
Type
Talk
Duration
20
Topic
Modern Software Development
Custom topic

Difficulty
Beginner
Abstract
Are you overwhelmed by organizing, sanitizing and prioritizing your issues?
Do you know “where” your issues are? Can you tell where your longrunners are? And why?
Do you need more detailed filter options in github?
Then it’s Triage Party Time!

Our teams at Red Hat’s AICoE were facing some challenges. We do Dev, Ops, and AI, and we have a diversity of features to implement, bugs to fix, and services to manage. The result is a pile of issues that make answering these questions hard. So we thought about a common process to get there.

Let us take you on our journey and show you how we:
Define the quality of an issue
What stages issues and PRs need to pass
How we plan things
How we measure progress
What tools we use

All of this while trying to make the process as natural and non-intrusive as possible, aiming to keep overhead to the minimum. We want it to feel more like a party than a burden.

To illustrate how fun this can be, we will do some real life issue refining and triage live on stage, so you can tell us if we managed to make this pleasant.

If you are involved in active projects hosted at github, come to this session to get some ideas on how to help manage them with less stress.
Could your session be a keynote?
No
Are you currently a student or have graduated in the past 6 months?
No
Do you consider yourself a member of one or more uderrepresented groups in technology?
No
Do you have a prior experience as a speaker at the conference?
Yes - meetups
If you have a recording of your previous speaking experience, share it with us.
N/A
Which region are you located in?
EMEA
My plans for 2022
In person
Additional information
The session would be co-presented with Thorsten Schwesig
I and all other secondary participants have read, and agree to all policies mentioned above.
Yes
</details>

**Comment by schwesig** [comment]
Accepted today, email to @codificat

<details><summary>On 22 Nov 2021 at 19:26, DevConf.CZ 2022 wrote:</summary>

> Dear Pep,
>
> tl;dr: Your proposal(s) have been accepted for DevConf.CZ 2022. Confirm your acceptance by Friday, December 3 at https://cfp.devconf.info/acceptproposals/. 
>
> We are very pleased to inform you that your proposal(s) listed below have been accepted for DevConf.CZ 2022.
>
> Overwhelmed by issues? Let's have a party! (20 minutes) for Modern Software Development track (where you're primary speaker)
>
> DevConf.CZ 2022 is going to be held on Jan 28-29, 2022 as a hybrid event with a shared in-person and virtual component during Central European Time. We are working hard to
> publish the schedule by December 15, 2021 using sched.org.
>
> Your talk/deep dive talk
>
> We are giving you the choice to speak in-person, do a live virtual presentation, or submit a pre-recorded presentation with live virtual Q&A. If you decide to pre-record your session, we
> need you to submit it to us by January 14 to give both you ample time to produce it and us ample time to review and upload it. To assist you with producing your talk, we are collecting
> a list of suggestions for both virtual and in-person speakers in the Speaker Guide: https://www.devconf.info/cz/speakerguide/. We also have a FAQ about all the things:
> https://www.devconf.info/cz/faq/.
>
> You have a fixed slot for giving your presentation. Be sure to include at least 5 minutes for questions from the audience. Feel free to recruit your colleagues to interact with people in
> chat while you are talking. A session chair will help you monitor the time spent during your time slot and assist you in case you run into any issues. We encourage you to join our
> discord channel to connect with your session chair, other speakers and organizers before the event: https://discord.gg/devconf
>
> Your lightning talk
>
> All lightning talks must be pre pre-recorded and submitted by January 14.
>
> Hybrid format
>
> The virtual component of the conference will be hosted on our event platform: https://hopin.com/ The in-person component of the conference will be held at FIT VUT in Brno, Czech
> Republic: https://www.fit.vut.cz/. We will be following the latest government advice on in-person events and working to ensure our onsite event is as Covid-safe and accessible as
> possible. We will keep you updated on what this means for the event throughout the planning process. Since event and travel conditions keep changing locally, we reserve the right to
> cancel the in-person component of the event due to Covid-related issues. We plan to make the final reevaluation of the hybrid format on December 22. We advise against
> non-refundable travel purchases before this date.
>
> Confirm your participation by December 3
>
> We have reviewed the submitted proposals and we are really interested in including your talk in the event. Please confirm your participation in DevConf.CZ 2022 by filling in this form
> https://cfp.devconf.info/acceptproposals/ as soon as possible, and no later than December 3. If we don't have confirmation from you by that date, your time slot will be allocated to
> another speaker. If you have any time preferences or limitations for your presentation or other concerns, please comment on the form linked above.
>
> We may have accepted more than one session that you proposed, we are expecting you to do at least one, but please don’t think you need to do them all. You can opt out from
> presenting any of the accepted talks by unticking the talk title in the form linked above. Again, please contact us with any questions.
>
> Important dates
>
> * Confirm speaking: December 3, 2021
> * Schedule published: December 15, 2021
> * Hybrid format reevaluation: December 22, 2021
> * Video recordings due: January 14, 2022
> * Event: January 28-29, 2022
>
> If you have any questions let us know at speakers@devconf.cz
>
> Please accept our congratulations on your proposal acceptance to DevConf.CZ 2022!
>
> We will be happy to see you there!
>
> DevConf.CZ 2022 Team
</details>

**Comment by codificat** [comment]
The conference is now over, and the talk was delivered.

Link to the session in the schedule: https://sched.co/siIG (slides are attached there)

Session recording: https://youtu.be/TK9ovKwj3xE

Thanks @schwesig for all your work and energy on this.
/close
